src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -18,7 +19,7 @@ public Driver create(Driver driver) {
 
     @Override
     public Driver get(Long id) {
-        return driverDao.get(id).get();
+        return driverDao.get(id).orElseThrow(NoSuchElementException::new);
```

Comment:
You should put message into this exception. Include value of 'id' into this message